friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import styled from "styled-components";
+
+const StyledForm = styled.div`
+ width: 100%;
+  h4 {
+    color:  #230C0F;
+    font-size: 2rem;
+    margin: 0.5rem 0;
+  }
+ form {
+  width: 60%;
+  display:block;
+  padding: 1rem;
+  margin: 0 auto;
+   input {
+     width: 100%;
+     padding: 1.5rem 0;
+     margin: 0.5rem 0;
+     border: 1px #A2D3C2 solid;
+     border-radius: 0.2rem;
+     text-align: center;
+   }
+   button {
+     width: 100%
+     border: none;
+     border-radius: 0.2rem;
+     padding: 1.5rem;
+     background-color: #A2D3C2;
+     color: #230C0F;
+   }
+ }
+`;
+
+export default function FriendForm(props) {
+  const [name, updateName] = useState("");
```

Comment:
You might want to create it like below

const [friend, setFriend] = useState({
   name: "",
   age: "",
   email: ""
})